src/apps/btc/btc_sign.c

```diff
@@ -433,7 +449,14 @@ app_btc_sign_error_t app_btc_sign_output(
         next_out->type = BTCSignNextResponse_Type_OUTPUT;
         next_out->index = _index;
     } else {
-        // Done with outputs. Verify total and fee.
+        // Done with outputs. Verify locktime, total and fee.
+        if (_locktime > 0 && _locktime_applies) {
+            // the RBF nsequence bytes only make sense in conjunction with a locktime,
```

Comment:
comment is not technically accurate, rbf makes sense independent of the locktime value.